### ISSUE TITLE
Estrutura do repositório

### ISSUE THREAD

**Issue by ivynasantino** [OpenDevUFCG/OpenDevUFCGBlog#11]
A estrutura atual do repositório deixa os posts "jogados" na primeira página, não seria mais interessante guardar os posts em pastas não?

Ex.: post/_area_, sendo as áreas, dados, infra, engenharia,..

**Comment by JuanBarros2** [comment]
Sim. Iremos organizar em pastas e manter um changelog para monter traqueado os temas que já foram tratados. Obrigado pela sugestão @ivynasantino !
